src/borg/archiver.py

```diff
@@ -1587,7 +1590,10 @@ def do_upgrade(self, args, repository, manifest=None, key=None):
     @with_repository(cache=True, exclusive=True, compatibility=(Manifest.Operation.CHECK,))
     def do_recreate(self, args, repository, manifest, key, cache):
         """Re-create archives"""
-        msg = ("recreate is an experimental feature.\n"
+        msg = ("This is a potentially dangerous functionality.\n"
+               "recreate might lead to data loss (if used wrongly or in case of bugs).\n"
```

Comment:
s/data loss/data loss in subtle ways/?

Because it seems obvious that using a borg command incorrectly can lead to data loss (e.g. borg delete). The key thing about recreate is that it can subtly lead to data loss if you accidentally exclude all the things.

Reply:
hmm, as a non-native english speaker, i guess i would not really get more informed when reading "in subtle ways".

not sure if it is due to my english skills or just not being more information there. :-)

Reply:
Me not being a native speaker either probably doesn’t help.

Adding the hint makes the warning much less "well, that’s kinda obvious" to me. Hopefully we can get some natives in!

Reply:
That "in subtle ways" makes the warning sound rather strange. Plus I would not call an `--exclude` that excludes everything subtle. The fact that `--exclude` excludes stuff is pretty obvious, its just that your pattern may not work as expected, much like with `prune`.